.clang-tidy

```diff
@@ -58,20 +58,17 @@ modernize-*,
 performance-*,
 portability-*,
 readability-*,
--readability-convert-member-functions-to-static,
--readability-else-after-return,
 -readability-function-cognitive-complexity,
--readability-identifier-length,
--readability-implicit-bool-conversion,
--readability-isolate-declaration,
--readability-make-member-function-const,
--readability-named-parameter,
 -readability-magic-numbers,
 -readability-redundant-access-specifiers,
 -readability-redundant-member-init,
 -readability-suspicious-call-argument
 '
 
+MinimumVariableNameLength = 0,
+MinimumParameterNameLength = 0,
+MinimumLoopCounterNameLength = 0,
+
```

Comment:
this is actually worse than keeping `-readability-identifier-length`. overriding the default options is mainly done when you agree with the check but disagree with its settings (e.g. you want to make the minimum 3 instead of 4, or 2 instead of 3, etc.), but not when you completely disagree with the check

alternatively, you could add a list of exceptions. the issue is that that list becomes quite long as well 😬